src/model.d.ts

```diff
@@ -2207,7 +2207,7 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
         [key in keyof Attributes<M>]?: Attributes<M>[key] | Fn | Col | Literal;
     },
     options: UpdateOptions<Attributes<M>>
-  ): Promise<[number, M[]]>;
+  ): Promise<[number] | [number, M[]]>;
```

Comment:
A few lines above this the docs for this method say:

> The promise returns an array with one or two elements...

Reply:
And I can confirm that the functionality is accurate based on the docs and that the types didn't match the return type which is why I'm submitting this PR.